src/utils/get-in.ts

```diff
@@ -1,24 +1,18 @@
-// This may look suspicious, but the point is to add ImmutableJS integration
-// only if the project itself already has a dependency on immutable. If not,
-// then this variable is left null and no integration is attempted.
-let immutable;
-try {
-  immutable = require('immutable');
-} catch (e) {}
-
 /*
  * Gets a deeply-nested property value from an object, given a 'path'
  * of property names or array indices.
  */
-export function getIn(
-  v: Object,
-  pathElems: (string | number)[]): any {
+export function getIn(v, pathElems: (string | number)[]): any {
     if (!v) {
       return v;
     }
 
-    if (immutable && immutable.Iterable.isIterable(v)) {
-      return (<{getIn: (path: (string | number)[]) => any}>v).getIn(pathElems);
+    // If this is an ImmutableJS structure, use existing getIn function
```

Comment:
My opinion is we are probably better off relying on a public API vs an internal implementation detail. My favorite option would be to call Irritable.isIterable but that created a lot of other require-related problems.